### PR TITLE
Implement Collatz Conjecture

### PR DESCRIPTION
**Pull request by jpreese**

### CODE REVIEW

File: exercises/collatz-conjecture/CollatzConjecture.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace CollatzConjecture
+{
+    public class CollatzConjecture
```
Comment (ErikSchierboom) [inline exercises/collatz-conjecture/CollatzConjecture.cs]: Could you make this a `static` class?

**Comment by jpreese** [comment]
All set. My [Spiral Matrix](https://github.com/exercism/csharp/blob/master/exercises/spiral-matrix/SpiralMatrix.cs) implementation didn't have it (won't add it here and muddle the PR), so I thought maybe I could get away with it on this one =P It does make sense though. I'll get you yet @ErikSchierboom !

**Comment by jpreese** [comment]
Looks like Travis got lazy and failed to fetch some dependencies. If you rerun the the build we should be good.

**Comment by ErikSchierboom** [comment]
@jpreese Yeah, I missed it in the Spiral Matrix version :) I have just restarted the Travis build. Once is succeeds, I'll merge this. Due to your hard work, there is only one [exercise left](http://exercism.io/languages/csharp/todo): Complex Numbers :)

**Comment by ErikSchierboom** [comment]
Thanks @jpreese! :tada:

**Pull request merged**